pkg/apis/autoscaling/v1alpha1/pa_lifecycle_test.go

```diff
@@ -1143,3 +1143,14 @@ func TestScaleToZeroPodRetention(t *testing.T) {
 		})
 	}
 }
+
+func TestInitiallyActive(t *testing.T) {
+	p := &PodAutoscaler{}
```

Comment:
```suggestion
	p := PodAutoscaler{}
```
will probably work :)